README.md

```diff
@@ -80,6 +80,12 @@ const Range = createSliderWithTooltip(Slider.Range);
 
 [Online demo](http://react-component.github.io/slider/examples/handle.html)
 
+After Range or Slider was wrapped by createSliderWithTooltip, it will have the following props:
+
+| Name         | Type    | Default | Description |
+| ------------ | ------- | ------- | ----------- |
+| tipFormatter | (value: number): React.ReactNode | value => value | A function to format tooltip overlay, if it's null, tooltip will be invisible |
```

Comment:
>  if it's null, tooltip will be invisible

This logic seems kind of weird，If one use `createSliderWithTooltip`, I think he wants to show tooltip.

`tipFormatter` is  just a way of data transform which default value is not transform